app/views/posts/index.html.erb

```diff
@@ -1 +1,46 @@
-<h1>This is the posts file for testing</h1>
\ No newline at end of file
+<div style="margin: 20px 40px;">
+  <div class="card mb-3" style="border: none;">
+    <div class="row g-0" style="gap: 1.5%;">
+      <div class="col-md-4" style="width: 70px;">
+        <img src ='<%= @user.photo %>' class="img-fluid rounded" alt="User Photo">
+      </div>
+      <div class="col-md-8">
+        <div class="card-body" style="border: 1px solid black; margin-top: 5px; width: 1345px;">
+          <h5 class="card-title"><%= @user.name %></h5>
+          <p class="card-text"><small class="text-muted">
+            Number of posts: <%= @user.posts_counter %></small>
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <% @posts.each do |post|%>
+    <div class="card border-success mb-3">
+      <div class="card-body text-success">
+        <a href = <%= user_post_path(post.author.id, post.id) %> style="text-decoration: none;">
+          <h5 class="card-title">Post <%= @posts.index(post) + 1%></h5>
+        </a>
+        <p class="card-text">
+          <%= post.text %>
+        </p>
+        <div class= "card-text" style="float: right">
+          <span class="card-text">Comments: <%= post.comments_counter%></span>
+          <span class="card-text">Likes: <%= post.likes_counter%></span>
+        </div>
+      </div>
+      <% if post.comments.any? %>
```

Comment:
- [ ] The comments that are rendered on the post index page should be the recent comments, not all comments. Kindly fix this.

Reply:
I have fixed this issue and recent comments are now visible in the post index page:

![5most_recent_comments](https://github.com/Zafron047/Blog_app_by_Rails/assets/117600135/26858435-8b15-49fa-9a48-59d3eadcc275)